packages/ui/src/lib/spinners/Spinner.stories.svelte

```diff
@@ -0,0 +1,25 @@
+<script>
+	import { Meta, Story } from '@storybook/addon-svelte-csf';
+
+	import Spinner from './Spinner.svelte';
```

Comment:
```suggestion
	import Spinner from './Spinner.svelte';
	import Button from "../button/Button.svelte";
```

This import would be needed if you accept my suggestion below.